src/sentry/static/sentry/app/utils/theme.js

```diff
@@ -72,6 +68,18 @@ const theme = {
       textDark: '#5d3e3b',
     },
   },
+
+  type: {
```

Comment:
is there a better name for this, `type` is generically used in a lot of places

Reply:
Oof, good call. I'll think of something better.